javascript-data-model/order-history.js

```diff
@@ -0,0 +1,72 @@
+var orderHistory = [
+  {
+    ordered: 'August 4, 2020',
+    total: '$34.00',
+    ship: 'JS Masher',
+    orderNumber: '114 - 3941689 - 8772232',
+    type: 'book',
+    title: 'Javascript for impatient programmers',
+    delivered: 'Aug 8, 2020',
+    reviewed: false,
+    buyAgain: false,
+    returnWindowClosed: true,
+    returnWindowDate: 'Sept 7, 2020',
+    itemPrice: '$31.55'
+  },
+  {
+    ordered: 'July 19, 2020',
+    total: '$44.53',
+    ship: 'JS Masher',
+    orderNumber: '113-9984268-1280257',
+    type: 'book',
+    title: 'The Timeless Way of Building',
+    delivered: 'Jul 20, 2020',
+    reviewed: false,
+    buyAgain: false,
+    returnWindowClosed: true,
+    returnWindowDate: 'Aug 19, 2020',
+    itemPrice: '$41.33'
+  },
+
+  {
+    ordered: 'July 4, 2020',
+    total: '$17.22',
+    ship: 'JS Masher',
+    orderNumber: '114-2875557-9059409',
+    type: 'console',
+    title: 'Gamecube Controller Adapter. Super Smash Bros Switch Gamecube  Adapter for WII UIEvent PC. Support Turbo and Vibration Feaures. No Driver and No Lag Gamecube Adapter',
+    delivered: 'Jul 07, 2020',
+    reviewed: false,
+    buyAgain: false,
+    returnWindowClosed: true,
+    returnWindowDate: 'Aug 05, 2020',
+    itemPrice: '$15.98'
+
+  },
+
+  {
+
+    ordered: 'July 3,2020',
+    total: '$138.93',
+    ship: 'JS Masher',
+    orderNumber: '113-2883177-2648248',
+    delivered: 'Jul 05,2020',
+    reviewed: false,
+    buyAgain: false,
+    returnedWindowClosed: true,
+    returnedWindowDate: 'Aug 04,2020',
+    items: [
+      {
+        type: 'controller',
+        title: 'GameCube Controller- Super Smash Bros. Edition (Nintendo Switch)',
+        itemPrice: '$94.95'
+      },
+      {
+        type: 'book',
+        title: 'The Art of Sql',
+        itemPrice: '$33.99'
+      }
+    ]
```

Comment:
This format doesn't match the format you used in your other order objects, please revise so that all the objects have the same format.

Reply:
hi cody.  updated and pushed with the same format.